packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -604,11 +644,33 @@ contract Deploy is Deployer {
         console.log("DisputeGameFactory version: %s", version);
     }
 
+    function initializeSuperchainConfig() public broadcast {
+        address payable superchainConfigProxy = mustGetAddress("SuperchainConfigProxy");
+        address superchainConfig = mustGetAddress("SuperchainConfig");
+
+        bytes32 batcherHash = bytes32(uint256(uint160(cfg.batchSenderAddress())));
+
+        Proxy(superchainConfigProxy).upgradeToAndCall({
+            _implementation: superchainConfig,
+            _data: abi.encodeCall(
+                SuperchainConfig.initialize,
+                (
+                    cfg.finalSystemOwner(), // systemOwner
+                    cfg.securityCouncil(), // initiator
```

Comment:
It feels like a bit of a leaky abstraction to have security council in the deploy config instead of initiator. Something that I'd like to strive for is consistency in the language. We already have some issues with this, like unsafe block signer vs p2p sequencer key. Since initiator is used in the contract, I'd prefer to keep that language in the deploy config. We can add a comment saying that in practice, it is the security council

Reply:
I've set these back to the more standard names